frontend/src/services/mocks/talks-en.json

```
@@ -0,0 +1,26 @@
+[
+  {
+      "uid": "123",
+      "title": "Introduction to Tournesol",

```

To ease the navigation, let's also add the key `name`.

```suggestion
     "name": "introduction_to_tournesol",
     "title": "Introduction to Tournesol",
```

This key doesn't contain space and will allow us to create HTML anchors. For instance:

 `https://tournesol.app/talk#introduction_to_tournesol` will refer to the Introduction to Tournesol talk.

---

good idea, it's done